Containerfile

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
```

Comment:
please remove copyright notices

Reply:
Sure, which raises a fair point; what copyright IS semonitor? Without any license, its copyright @jbuehl all rights reserved, which makes contributions interesting? Maybe open a seperate issue for this?

Reply:
> what copyright IS semonitor?

Heh, good question

Reply:
I just added the GPL.

Reply:
I'll leave the SPDX license identifier out per your request even still :) (though it is very common to mark source files with it :)